etc/profile-m-z/retroarch.profile

```diff
@@ -0,0 +1,46 @@
+# Persistent local customizations
+include retroarch.local
+# Persistent global definitions
+include globals.local
+
+blacklist /boot
+blacklist /opt
+blacklist /usr/libexec
+blacklist /var
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+whitelist ${HOME}/.config/retroarch
```

Comment:
`mkdir`+`blacklist`(in dp.inc)+`noblacklist`

Reply:
????

Reply:
https://github.com/netblue30/firejail/blob/07e9029b24e21de4c6ecae3060bed3d2450bb296/etc/templates/profile.template#L62-L64

```suggestion
mkdir ${HOME}/.config/retroarch
whitelist ${HOME}/.config/retroarch
```

Reply:
I don't understand why `whitelist` requires `mkdir`, but I just added it.